jdk-1.6-parent/select2-parent/select2/src/main/java/org/wicketstuff/select2/AbstractSelect2Choice.java

```diff
@@ -54,9 +55,11 @@
 	private List<T> choices;
 	private ChoiceRenderer<T> renderer;
 
+	private transient boolean convertValuePerformed = false;

```

Comment:
No need to be `transient`. The first is reset to `false` in `#onDetach()`.
Additionally: the page is stored in the HttpSession and the next request may read it as a live instance from there, so `transient` won't help. It will help only if the page is deeper in the history stack, i.e. in the disk store, serialized as byte[].